lib/workers/repository/process/deprecated.js

```diff
@@ -53,5 +56,20 @@ async function raiseDeprecationWarnings(config, packageFiles) {
         await platform.ensureIssue(issueTitle, issueBody, ensureOnce);
       }
     }
+    logger.debug(
+      'Closing existing depricated package issues missing in current deprecatedPackages'
+    );
+    const issueList = await platform.getIssueList();
+    const titleMatch = 'Dependency deprecation warning';
```

Comment:
Find a way to share this string with the earlier code for creating issues that also uses the same (line 44)